ifrs17-template/Test/TestData.ipynb

```diff
@@ -0,0 +1,366 @@
+                "\n                                                        Scenario = scenario, ",
+                "\n                                                        Year = args.Year,",
+                "\n                                                        Month = args.Month };",
+                "\nvar previousPeriodPartition = new PartitionByReportingNodeAndPeriod { Id = (Guid)(await DataSource.Partition.GetKeyForInstanceAsync<PartitionByReportingNodeAndPeriod>(previousArgs)),",
+                "\n                                                                      ReportingNode = reportingNode, ",
+                "\n                                                                      Scenario = scenario, ",
+                "\n                                                                      Year = previousArgs.Year,",
+                "\n                                                                      Month = previousArgs.Month };",
+                "\nvar partitionReportingNode = new PartitionByReportingNode { Id = (Guid)(await DataSource.Partition.GetKeyForInstanceAsync<PartitionByReportingNode>(args)),",
+                "\n                                                        ReportingNode = args.ReportingNode};"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# DataNodes and DataNodeData"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var dt1 = new Portfolio(){",
+                "\n    Partition = partitionReportingNode.Id,",
+                "\n    ContractualCurrency = \"USD\",",
+                "\n    LineOfBusiness = \"ANN\",",
+                "\n    ValuationApproach =\"BBA\",",
+                "\n    OciType = \"Default\",",
+                "\n    SystemName = \"DT1\",",
+                "\n    DisplayName = \"DT1 OCI\"",
+                "\n};",
+                "\nvar dt11 = new GroupOfInsuranceContract(){",
+                "\n    Portfolio = \"DT1\",",
+                "\n    Profitability = \"P\",",
+                "\n    LiabilityType = \"LRC\",",
+                "\n    AnnualCohort = 2020,",
+                "\n    Partition = partitionReportingNode.Id,",
+                "\n    ContractualCurrency =  \"USD\",",
+                "\n    LineOfBusiness = \"ANN\",",
+                "\n    ValuationApproach = \"BBA\",",
+                "\n    OciType =\"Default\" ,",
+                "\n    SystemName = \"DT1.1\",",
+                "\n    DisplayName = \"DT1.1 OCI LRC PA 0.8\"",
+                "\n};",
+                "\n",
+                "\nvar dtr11 = new GroupOfReinsuranceContract(){",
+                "\n    Portfolio = \"DT1\",",
```

Comment:
Group of Reinsurance Contracts belong to Reinsurance Portfolio.
Here should be DTR1 which needs to be created

Reply:
Created